contracts/rewards/base/emission_manager.cairo

```diff
@@ -41,6 +43,7 @@ end
 @external
 func configure_assets{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}():
     # TODO
+    # Needs an oracle.
```

Comment:
cool! I've discussed with Stork's team yesterday (https://medium.com/@stork-oracle/stork-meet-the-starknet-native-oracle-8409a129010d) and they have built an oracle for starknet. it's currently deployed on the testnet. we can try to integrate it as a next task